server/memo_comment.go

```diff
@@ -0,0 +1,190 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"github.com/usememos/memos/api"
+	"github.com/usememos/memos/common"
+	"github.com/usememos/memos/store"
+)
+
+func (s *Server) registerMemoCommentRoutes(g *echo.Group) {
```

Comment:
Please move them into `/api/v1`. e.g. https://github.com/usememos/memos/pull/1842

Reply:
ok.

Reply:
move `registerMemoCommentRoutes` to `api/v1`. For example, https://github.com/usememos/memos/blob/main/api/v1/resource.go#L86

Reply:
ok,finished